<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileAdd</class>
 <widget class="QWidget" name="FileAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить</string>
  </property>
  <widget class="QPushButton" name="pushButtonFileAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя файла</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSizeAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Размер</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserFileAdd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; text-decoration: underline;&quot;&gt;Добавление файла&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLocationAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Путь</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOwnerAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Владелец(ID)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNameAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSizeAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLocationAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOwnerAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <zorder>textBrowserFileAdd</zorder>
  <zorder>pushButtonFileAdd</zorder>
  <zorder>labelNameAdd</zorder>
  <zorder>labelSizeAdd</zorder>
  <zorder>labelLocationAdd</zorder>
  <zorder>labelOwnerAdd</zorder>
  <zorder>lineEditNameAdd</zorder>
  <zorder>lineEditSizeAdd</zorder>
  <zorder>lineEditLocationAdd</zorder>
  <zorder>lineEditOwnerAdd</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
